__tests__/app.test.js

```diff
@@ -257,4 +257,144 @@ describe("GET /api/articles/:article_id/comments", () => {
+    test("responds with correct article with an updated vote property when the increment is positive", () => {
+
+        const update = { inc_votes: 1 }
+
+        return request(app)
+        .patch('/api/articles/3')
+        .send(update)
+        .expect(200)
+        .then((article) => {
+
+            expect(article._body).toHaveProperty('article_id', 3 )
+            expect(article._body).toHaveProperty('title', expect.any(String))
+            expect(article._body).toHaveProperty('topic', expect.any(String))
+            expect(article._body).toHaveProperty('author', expect.any(String))
+            expect(article._body).toHaveProperty('body', expect.any(String))
+            expect(article._body).toHaveProperty('created_at', expect.any(String))
+            expect(article._body).toHaveProperty('votes', 1)
+            expect(article._body).toHaveProperty('article_img_url', expect.any(String))
+        })
+
+    })
+
+
+    test("responds with correct article with an updated vote property when the increment is negative", () => {
+
+        const update = { inc_votes: -10 }
+
+        return request(app)
+        .patch('/api/articles/3')
+        .send(update)
+        .expect(200)
+        .then((article) => {
+
+            expect(article._body).toHaveProperty('article_id', 3 )
+            expect(article._body).toHaveProperty('title', expect.any(String))
+            expect(article._body).toHaveProperty('topic', expect.any(String))
+            expect(article._body).toHaveProperty('author', expect.any(String))
+            expect(article._body).toHaveProperty('body', expect.any(String))
+            expect(article._body).toHaveProperty('created_at', expect.any(String))
+            expect(article._body).toHaveProperty('votes', -10)
+            expect(article._body).toHaveProperty('article_img_url', expect.any(String))
+        })
+
+    })
+
+    test("responds with correct article with an updated vote property and ignores any invalid properties on the request body", () => {
+
+        const update = { inc_votes: -10,
+                         ignorableProperty : 'please ignore me'}
+
+        return request(app)
+        .patch('/api/articles/3')
+        .send(update)
+        .expect(200)
+        .then((article) => {
+
+            expect(article._body).toHaveProperty('article_id', 3 )
+            expect(article._body).toHaveProperty('title', expect.any(String))
+            expect(article._body).toHaveProperty('topic', expect.any(String))
+            expect(article._body).toHaveProperty('author', expect.any(String))
+            expect(article._body).toHaveProperty('body', expect.any(String))
+            expect(article._body).toHaveProperty('created_at', expect.any(String))
+            expect(article._body).toHaveProperty('votes', -10)
+            expect(article._body).toHaveProperty('article_img_url', expect.any(String))
+        })
+
+    })
+
+
+    test("returns a 400 when sent an invalid type (not a number)", () => {
```

Comment:
I would be clearer with this description - when article_id is not a valid type? Just because you are also 'sending' an update